buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -74,7 +74,9 @@ pipeline {
                 	withEnv(["JAVA_HOME=$WORKSPACE/openjdkbinary/j2sdk-image/${(params.JAVA_VERSION == 'SE80') ? 'jre/' : ''}",
                 	"JAVA_BIN=$WORKSPACE/openjdkbinary/j2sdk-image/${(params.JAVA_VERSION == 'SE80') ? 'jre/' : ''}bin",
                 	"JAVA_VERSION=${params.JAVA_VERSION}"]) {
-                		sh "$OPENJDK_TEST/maketest.sh $OPENJDK_TEST ${params.TARGET}"
+						wrap([$class: 'Xvfb']) {
```

Comment:
since we are using `DISPLAY=:1`, we need to tell Xvfb to set display name to `:1`.
This would be like `wrap([$class: 'Xvfb', displayName: 1]) {`